gcc/testsuite/rust.test/compile/block_expr4.rs

```diff
@@ -1,8 +1,40 @@
-fn foo() -> isize {
+fn foo() -> i32 {
```

Comment:
Can you make new test cases instead of amending existing ones. I would prefer to keep these test cases as they are.